documentation/publish/distribute-sideloading.md

```diff
@@ -117,9 +117,10 @@ To use Install Add-on From File in Add-on manager, send the user the signed add-
 
 {% capture content %}
 
+<section id="standard-extensions-folder"></section>
```

Comment:
section isn't needed here since the id for this h2 is already "standard-extension-folders"

Reply:
oof, should have caught that. thanks!